modules/virt-creating-localnet-nad-cli.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-creating-localnet-nad-cli_{context}"]
+= Creating a NAD for localnet topology using the CLI
+
+You can create a network attachment definition (NAD) which describes how to attach a pod to the underlying physical network.
+
+.Prerequisites
+* You have access to the cluster as a user with `cluster-admin` privileges.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the Kubernetes NMState Operator.
+* You have created a `NodeNetworkConfigurationPolicy` object to map the OVN-Kubernetes secondary network to an Open vSwitch (OVS) bridge.
+
+.Procedure
+
+. Create a `NetworkAttachmentDefinition` object:
++
+[source,yaml]
+----
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  name: localnet-network
+  namespace: default
+spec:
+  config: |2
+    {
+            "cniVersion": "0.3.1", <1>
+            "name": "localnet-network", <2>
+            "type": "ovn-k8s-cni-overlay", <3>
+            "topology": "localnet", <4>
+            "netAttachDefName": "default/localnet-network" <5>
+    }
+----
+<1> The CNI specification version. The required value is `0.3.1`.
+<2> The name of the network. This attribute must match the value of the `spec.desiredState.ovn.bridge-mappings.localnet` field of the `NodeNetworkConfigurationPolicy` object that defines the OVS bridge mapping.
+<3> The name of the CNI plug-in to be configured. The required value is `ovn-k8s-cni-overlay`.
+<4> The topological configuration for the network. The required value is `localnet`.
+<5> The value of the `namespace` and `name` fields in the `metadata` stanza of the `NetworkAttachmentDefinition` object.
```

Comment:
At least three of these five items have required values. What happens if a different value is inserted? Will the process fail to run? Will it be rejected? Will there be a message?

I'd address this in an admonition.

Reply:
The required values are different based on the topology. For localnet topology, the above values are required to create the network attachment definition. @maiqueb What happens if a different value is inserted?

Reply:
- cniVersion: it can also be 0.4.0 and nothing bad would happen. Cannot be 1.0.0 or larger. Unsure about what happens if you use 0.2.0 or 0.1.0. 
- type: must be the one presented or you would be using a different type of plugin that would not understand the other parameters. It would result in errors.
- topology: it must be localnet, or you'd be using a different topology, thus configuring a layer2 disconnected overlay instead of a network with access to the physical network